src/AutomationTests/AutomationIntegrationTests.cs

```diff
@@ -59,6 +60,41 @@ public void Scan_Integration()
             // Validate the output file exists where it is expected
             Assert.IsTrue(Regex.IsMatch(output.OutputFile.A11yTest, regexForExpectedFile));
             Assert.IsTrue(File.Exists(output.OutputFile.A11yTest));
+
+            EnsureGeneratedFileIsReadableByOldVersionsOfAxeWindows(output, TestProcess.Id);
+        }
+
+        private void EnsureGeneratedFileIsReadableByOldVersionsOfAxeWindows(ScanResults scanResults, int processId)
+        {
+            string targetFolder = Path.GetFullPath(
+                Path.Combine(Directory.GetCurrentDirectory(), @"..\..\..\CurrentFileVersionCompatibilityTests\bin",
```

Comment:
Other paths in this class are pulled out as readonly at the top of the class. Might be worth considering here.

Reply:
Done!